pkg/controller/scyllacluster/resource_test.go

```diff
@@ -1085,6 +1085,28 @@ func TestStatefulSetForRack(t *testing.T) {
 			}(),
 			expectedError: nil,
 		},
+		{
+			name: "new StatefulSet with non-empty externalSeeds in scylla container",
+			rack: newBasicRack(),
+			scyllaCluster: func() *scyllav1.ScyllaCluster {
+				sc := newBasicScyllaCluster()
+				sc.Spec.ExternalSeeds = []string{"10.0.1.1", "10.0.1.2", "10.0.1.3"}
+				return sc
+			}(),
+			existingStatefulSet: nil,
+			expectedStatefulSet: func() *appsv1.StatefulSet {
+				sts := newBasicStatefulSet()
+
+				for i := range sts.Spec.Template.Spec.Containers {
+					if sts.Spec.Template.Spec.Containers[i].Name == naming.ScyllaContainerName {
```

Comment:
What would you suggest? I'd rather avoid explicitly setting `sts.Spec.Template.Spec.Containers` field, since that adds a lot of bloat. Would a `panic` on not found scylla container be fine?

Reply:
done, hardcoded